### ISSUE TITLE
Timber::get_posts not accepting an array of post types

### ISSUE THREAD

**Issue by chrisgherbert** [timber/timber#935]
### Expected behavior

The first parameter of Timber::get_posts() should accept a standard array of WP_Query arguments - including an array of post types for `post_type`.  This should work:

```php
Timber::get_posts(array(
  'post_type' => array('post', 'page')
));
```

### Actual behavior

```php
PHP warning:  class_exists() expects parameter 1 to be string, array given in /path/to/project/vendor/timber/timber/lib/timber-post-getter.php on line 120
```
### Steps to reproduce behavior

```php
Timber::get_posts(array('post_type' => array('post', 'page')));
```

Adding an is_array check before the attempt to use the `post_type` value as a PHP class seems to fix it:

```php
		if (is_array($type)){
			return $type;
		}

		if (class_exists($type) && is_subclass_of($type, 'TimberPost')) {
			return $type;
		}
```
### What version of WordPress, PHP and Timber are you using?

WordPress 4.4.2, PHP 5.6.10, Timber 1.0-rc2

### How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)

Installed from Composer

**Comment by connorjburton** [comment]
Oh, this is an interesting one. Related issue #799 - I will take a look at this after `1.0.0`

**Comment by jono-hayward** [comment]
I'm having this issue too, but only since updating Wordpress to 4.5.1 and Timber to 1.0.0. Previously, my query worked perfectly with no errors:

    $context['latests'] = Timber::get_posts(array(
        'post_type' => array( 'checkpoint_news', 'checkpoint_review' ),
        'posts_per_page' => 5
    ));

Interestingly, this query still seems to work - I get an array of the latest five news posts and reviews. The difference is there's now a PHP error at the top of the page.

**Comment by connorjburton** [comment]
This will be fixed in `1.0.1` - sorry about this!

**Comment by angtheod** [comment]
Hello @connorjburton @jarednova  , didn't know if I should create a new issue, sorry before hand.
I wonder if this bug is solved?
If I give no arg to get_posts it does include attachments.. but if I supply an array as argument then... it does not include attachments...I have tried all possible combinations trying to include attachments in the get_posts results.. but no lack so far..

`$context['posts'] = Timber::get_posts( [

's'       => $search,
'post_type' => [
'post',
'page',
'attachment',
],
'post_status' => 'any',
'orderby' => 'post_date',
'order'   => 'DESC',
] );`
